rllib/env/multi_agent_env.py

```diff
@@ -33,6 +33,11 @@ class MultiAgentEnv(gym.Env):
     Agents are identified by (string) agent ids. Note that these "agents" here
     are not to be confused with RLlib Algorithms, which are also sometimes
     referred to as "agents" or "RL agents".
+
+    The preferred format for action- and observation space is a mapping from agent
```

Comment:
"The preferred format for action- and observation space is a mapping" Any kind of mapping? Like a dict?

Reply:
@cool-RR Yes, a dict!

Reply:
It should probably say dict then.